prometheus/exporter/network.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import subprocess
+import json
+import sys
+import re
+import time
+import logging  
+logger = logging.getLogger("gpu_expoter")  
+
+def convert_to_byte(data):
+    number = float(re.findall(r"(\d+(\.\d+)?)", data)[0][0])
+    if "T" in data:
+        return number * 1024 * 1024 * 1024 * 1024
+    elif "G" in data:
+        return number * 1024 * 1024 * 1024
+    elif "M" in data:
+        return number * 1024 * 1024
+    elif "K" in data:
+        return number * 1024
+    else: 
+        return number
+
+def parse_iftop(stats):
+    connectionDic = {}
+    data = [line.split(',') for line in stats.splitlines()]
+
+    while True and len(data) > 0:
```

Comment:
isn't this same with `while len(data) > 0`?

Reply:
good suggestion, will update it